eatngo-infra/postgresql/src/main/resources/application.yml

```
@@ -0,0 +1,20 @@
+  datasource:
+    url: ${PG_URL}
+    username: eatngo_user
+    password: ${PG_PASSWORD}
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)